core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -165,6 +167,7 @@
   private static final AtomicLong CALL_ID_COUNTER = new AtomicLong();
   // Election timeout to use in a single master cluster.
   private static final long SINGLE_MASTER_ELECTION_TIMEOUT_MS = 500;
+  private static final String UNDEFINED_LEADER_ID = "UNDEFINED_LEADER_ID";
```

Comment:
nit: When do you expect to see this ID? If this is because the masters are re-electing, maybe this can be changed to something like "WAITING FOR ELECTION" to be more expressive?

Reply:
Yeah, accept this great suggestion.